distributed/tests/test_scheduler.py

```diff
@@ -1983,10 +1983,10 @@ def abort(self):
         pass
 
     def read(self, deserializers=None):
-        raise EnvironmentError
+        raise OSError
 
     def write(self, msg, serializers=None, on_error=None):
-        raise EnvironmentError
+        raise OSError
```

Comment:
```suggestion
        raise OSError()
```

Reply:
out of curiosity, is this a style note or is there a functional difference?

Reply:
it's only a style note. ``except OSError as e`` will set e to an instance to OSError in both cases.